src/main/resources/admin/widgets/matomo/matomo.js

```diff
@@ -0,0 +1,79 @@
+const libs = {
+  httpClient: require("/lib/http-client"),
+  portal: require("/lib/xp/portal"),
+  thymeleaf: require("/lib/thymeleaf"),
+  content: require("/lib/xp/content"),
+};
+
+const settings = {
+  responseFormat: "JSON"
+}
+
+const token = app.config['matomo.token'];
```

Comment:
Denne eksisterer for alle som benytter matomo? Og heter matomo.token?

Reply:
@andersnin 😇

Reply:
Kommer en readme. Takk for påminnelse!